### PR TITLE
Run critest in parallel

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:
Make the critest suite succeed faster by running them in parallel.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: test/critest.bats [hunk 1]
```diff
@@ -19,6 +19,11 @@ function teardown() {
 	critest \
 		--runtime-endpoint "unix://${CRIO_SOCKET}" \
 		--image-endpoint "unix://${CRIO_SOCKET}" \
+		--parallel="$(nproc)" \
```
Comment (kwilczynski) [inline test/critest.bats:22]: Do we want to make it **-1** to keep some CPU for the rest of the things running and servicing I/O?

**Comment by kwilczynski** [comment]
/hold

For the review comment.

File: test/critest.bats (hunk 1 above)
Comment (saschagrunert) [reply test/critest.bats:22]: Hm, I don't think the tests use that much CPU so I'd like to keep it simple :)

**Pull request merged**